src/system/bus/i2c/i2cDevice.cpp

```diff
@@ -39,7 +39,6 @@ uint8_t I2CDevice::readByte(uint8_t regNum)
 bool I2CDevice::writeWord(uint16_t word)
 {
     uint8_t error = 0;
-    uint8_t* bytes = (uint8_t*)&word;
```

Comment:
took advantage of this commit to remove an unused variable that ws causing a warning